recipe/meta.yaml

```
@@ -21,101 +21,18 @@ requirements:
-    - langsmith >=0.1.0,<0.2.0
-  run_constrained:
-    - tqdm >=4.48.0
-    - openapi-pydantic >=0.3.2,<0.4.0
-    - faiss-cpu >=1.0.0,<2.0.0
-    - beautifulsoup4 >=4.0.0,<5.0.0
-    - jinja2 >=3.0.0,<4.0.0
-    - cohere >=4.0.0,<5.0.0
-    - openai <2
-    - arxiv >=1.4.0,<2.0.0
-    - pypdf >=3.4.0,<4.0.0
-    - aleph-alpha-client >=2.15.0,<3.0.0
-    - gradientai >=1.4.0,<2.0
-    - pgvector >=0.1.6,<0.2.0
-    - atlassian-python-api >=3.36.0,<4.0.0
-    - html2text >=2020.1.16,<2021.0.0
-    - numexpr >=2.8.6,<3.0.0
-    - jq >=1.4.1,<2.0.0
-    - pdfminer-six >=20221105.0.0,<20221106.0.0
-    - lxml >=4.9.3,<6.0.0
-    - pymupdf >=1.22.3,<2.0.0
-    - rapidocr-onnxruntime >=1.3.2,<2.0.0
-    - pypdfium2 >=4.10.0,<5.0.0
-    - gql >=3.4.1,<4.0.0
-    - pandas >=2.0.1,<3.0.0
-    - telethon >=1.28.5,<2.0.0
-    - chardet >=5.1.0,<6.0.0
-    - requests-toolbelt >=1.0.0,<2.0.0
-    - scikit-learn >=1.2.2,<2.0.0
-    - py-trello >=0.19.0,<0.20.0
-    - bibtexparser >=1.4.0,<2.0.0
-    - pyspark >=3.4.0,<4.0.0
-    - mwparserfromhell >=0.6.4,<0.7.0
-    - mwxml >=0.3.3,<0.4.0
-    - esprima-python >=4.0.1,<5.0.0
-    - streamlit >=1.18.0,<2.0.0
-    - psychicapi >=0.8.0,<0.9.0
-    - cassio >=0.1.0,<0.2.0
-    - sympy >=1.12.0,<2.0.0
-    - rapidfuzz >=3.1.1,<4.0.0
-    - jsonschema >1
-    - rank-bm25 >=0.2.2,<0.3.0
-    - geopandas >=0.13.1,<0.14.0
-    - gitpython >=3.1.32,<4.0.0
-    - feedparser >=6.0.10,<7.0.0
-    - newspaper3k >=0.2.8,<0.3.0
-    - xata >=1.0.0a7,<2.0.0
-    - xmltodict >=0.13.0,<0.14.0
-    - markdownify >=0.11.6,<0.12.0
-    - assemblyai >=0.17.0,<0.18.0
-    - sqlite-vss >=0.1.2,<0.2.0
-    - motor >=3.3.1,<4.0.0
-    - timescale-vector >=0.0.1,<0.0.2
-    - typer >=0.9.0,<0.10.0
-    - anthropic >=0.3.11,<0.4.0
-    - aiosqlite >=0.19.0,<0.20.0
-    - rspace_client >=2.5.0,<3.0.0
-    - upstash-redis >=0.15.0,<0.16.0
-    - google-cloud-documentai >=2.20.1,<3.0.0
-    - fireworks-ai >=0.9.0,<0.10.0
-    - javelin-sdk >=0.1.8,<0.2.0
-    - hologres-vector >=0.0.6,<0.0.7
-    - praw >=7.7.1,<8.0.0
-    - msal >=1.25.0,<2.0.0
-    - databricks-vectorsearch >=0.21.0,<0.22.0
-    - cloudpickle >=2.0.0
-    - dgml-utils >=0.3.0,<0.4.0
-    - datasets >=2.15.0,<3.0.0
-    - tree-sitter >=0.20.2,<0.21.0
-    - tree-sitter-languages >=1.8.0,<1.9.0
-    - azure-ai-documentintelligence >=1.0.0b1,<2.0.0
-    - oracle-ads >=2.9.1,<3.0.0
-    - httpx >=0.24.1,<0.25.0
-    - elasticsearch >=8.12.0,<9.0.0
-    - hdbcli >=2.19.21,<3.0.0
-    - oci >=2.119.1,<3.0.0
-    - rdflib 7.0.0
-    - nvidia-riva-client >=2.14.0,<3.0.0
-    - azure-search-documents 11.4.0
-    - azure-identity >=1.15.0,<2.0.0
-    - tidb-vector >=0.0.3,<1.0.0
-    - friendli-client >=1.2.4,<2.0.0
-    - premai >=0.3.25,<0.4.0
-    - vdms >=0.0.20,<0.0.21
-    - httpx-sse >=0.4.0,<0.5.0
-    - pyjwt >=2.8.0,<3.0.0
-    - oracledb >=2.2.0,<3.0.0
+    - pydantic-settings >=2.4.0,<3.0.0
+    - langsmith >=0.1.125,<0.3
+    - httpx-sse >=0.4.0
```

https://github.com/langchain-ai/langchain/blob/langchain-community%3D%3D0.3.12/libs/community/pyproject.toml#L43

```suggestion
    - httpx-sse >=0.4.0,<0.5.0
```